internal/nodes/avi_vrf_translator.go

```diff
@@ -97,10 +97,17 @@ func (o *AviObjectGraph) BuildVRFGraph(key, vrfName, nodeName string, deleteFlag
 			diff := lenNewNodeRoutes - nodeStaticRouteDetails.Count
 
 			var staticRouteCopy []*models.StaticRoute
-			staticRouteCopy = append(staticRouteCopy, aviVrfNode.StaticRoutes[:startIndex]...)
+			if startIndex > -1 && startIndex <= len(aviVrfNode.StaticRoutes) {
```

Comment:
the startIndex is non negative, so can we change this comparison to >=0. also can we change the type for startIndex to unsigned int?